content/developer/reference/cli.rst

```diff
@@ -82,6 +82,11 @@ Running the server
     comma-separated list of modules to update before running the server.
     Use ``all`` for all modules. (requires :option:`-d`).
 
+.. option:: --reinit <modules>
+
+    comma-separated list of modules to reinitialize before running the server
+    (requires :option:`-d`).
+
```

Comment:
I would add a paragraph to explain how this option is different than `-u`, and maybe update the entry for `-u` to explain the difference with `--reinit`. I would also emphasis that `--reinit` is a development option that should not be used in production, along with the risks of using it in production (like to change any manual configuration done in the ir.rules, if I did correctly understand the purpose of the command).